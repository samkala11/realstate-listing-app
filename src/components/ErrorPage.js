import React from 'react';


const ErrorPage = () => {
    return (
        <h3> 404 - Not found </h3>
    )
}

export default ErrorPage;