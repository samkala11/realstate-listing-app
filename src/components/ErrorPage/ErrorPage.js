import React from 'react';

const ErrorPage = () => {
    return (
        <div  className="error-page-title" data-testid="error-page-title" > 404 - Not found </div>
    )
}

export default ErrorPage;