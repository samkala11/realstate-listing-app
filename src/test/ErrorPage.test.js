import { render, screen, cleanup } from '@testing-library/react';
import ErrorPage from '../components/ErrorPage/ErrorPage';
import renderer from 'react-test-renderer';


afterEach(cleanup);


test('Renders Error page successfully', () => {

    render(<ErrorPage />);

    const errorPageTitle = screen.getByTestId('error-page-title');
    expect(errorPageTitle).toBeInTheDocument();
  
});

test('Error page matches snapshot', () => {
    const tree = renderer.create(<ErrorPage />).toJSON();
    expect(tree).toMatchSnapshot();
})