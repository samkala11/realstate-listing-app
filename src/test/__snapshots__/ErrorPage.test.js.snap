// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error page matches snapshot 1`] = `
<div
  className="error-page-title"
  data-testid="error-page-title"
>
   404 - Not found 
</div>
`;
